src/Components/Institutions/Institutions.jsx

```diff
@@ -19,6 +19,10 @@ const dummyData = [
     insti: "Indian Institute of Technology, Bombay",
     pfp: dp,
   },
+  {
+    insti: "Indian Institute of Technology, Bombay",
+    pfp: dp,
+  },
 ];
```

Comment:
Use all the data from a json file in public folder

for example if there is a json file, in the path
/public/db/institute.json

Use fetch API to fetch the data in the Page 

```js 
fetch("/db/institute.json")
```

Reply:
Implemented fetch API.